impl/parquet_writer.go

```diff
@@ -26,10 +28,14 @@ func NewParquetWriter(fileWriter source.ParquetFile) (*ParquetWriter, error) {
 }
 
 type ParquetMessage struct {
-	Value   string `parquet:"name=value, type=BYTE_ARRAY, convertedtype=UTF8, encoding=PLAIN"`
-	Topic   string `parquet:"name=topic, type=BYTE_ARRAY, convertedtype=UTF8, encoding=PLAIN"`
-	Key     string `parquet:"name=key, type=BYTE_ARRAY, convertedtype=UTF8, encoding=PLAIN"`
-	Headers string `parquet:"name=headers, type=BYTE_ARRAY, convertedtype=UTF8, encoding=PLAIN"`
+	Value         string `parquet:"name=value, type=BYTE_ARRAY, convertedtype=UTF8, encoding=PLAIN"`
+	Topic         string `parquet:"name=topic, type=BYTE_ARRAY, convertedtype=UTF8, encoding=PLAIN"`
+	Partition     int32  `parquet:"name=partition, type=INT32, convertedtype=INT_32"`
+	Offset        string `parquet:"name=offset, type=BYTE_ARRAY, convertedtype=UTF8, encoding=PLAIN"`
```

Comment:
Sometimes, I utilize the streaming feature to transition from one topic to another with more partitions. This is why I don't include the partition and offset in the message.
I prefer using it as an option.

Reply:
ok will make it as an option then.

Reply:
Sorry for the late response. I think it would be better if we always store the partition and offset and make writing optional. What do you think about this approach?

Reply:
Yeah, it make sense!! will do the changes

Reply:
done